examples/helpers/mod.rs

```diff
@@ -0,0 +1,45 @@
+#[cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))]
+extern crate smithay_client_toolkit as sctk;
+
+#[cfg(any(target_os = "linux", target_os = "dragonfly", target_os = "freebsd", target_os = "netbsd", target_os = "openbsd"))]
+use winit::os::unix::WindowExt;
+
+// Wayland requires the commiting of a surface to display a window
+pub fn init_wayland(window: &winit::Window) {
```

Comment:
This is emitting `unused_variables` warnings on non-linux platforms. Could you add a `let _ = window;` line somewhere to suppress that warning?

Reply:
For clarity, I'd actually rather move the `#[cfg(...)]` in the call site in the examples, so that people can clearly see that this code is for some platform-specific stuff.